src/BursarExportPlugin.test.tsx

```diff
@@ -0,0 +1,97 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
too many such lint errors :(

Reply:
For this one, should I list '@testing-library/react'  in the project's dependencies. This is an error that shows up in all files importing from there.

Reply:
@danetsao you should include `@testing-library/react`.  For stripes-components, we should never use `@folio/stripes-components` directly; all should be through `@folio/stripes/components`

Reply:
@danetsao you need to use `'@folio/jest-config-stripes/testing-library/react'` instead of `'@testing-library/react'`.